plugins/vpp/ifplugin/data_resync.go

```diff
@@ -69,6 +69,13 @@ func (c *InterfaceConfigurator) Resync(nbIfs []*intf.Interfaces_Interface) error
 			}
 			continue
 		}
+		// Register IPSec tunnel interfaces, but do not resync them (they are resynced in IPSec plugin)
+		if vppIf.Interface.Type == intf.InterfaceType_IPSEC_TUNNEL {
+			if err := c.registerInterface(vppIf.Meta.InternalName, vppIfIdx, vppIf.Interface); err != nil {
+				return errors.Errorf("interface resync error: %v", err)
```

Comment:
This should most likely mention that it failed to register ipsec tunnel interface.

Reply:
registration itself will not fail but there is also a binary API storing interface tag/index which could fail, and it also returns the proper message